### PR TITLE
Feature/timetable user not registered

### PR DESCRIPTION
**Pull request by andrejskalnins**

### CODE REVIEW

**Review by Abdalrazaq94 — DISMISSED** [review]
client/src/pages/Timetables/TimetableNotRegistered.js 
in this file, you need to delete the import FooterNav and Footer Icon and Footer Logo.

and in the end of the code delete 
<FooterLogo />
			<FooterNav />
			<FooterIcon />
			
			
			as we have changed already, otherwise you will have conflict

File: client/src/pages/Timetables/TimetableNotRegistered.css [hunk 1]
```diff
@@ -0,0 +1,255 @@
+.timetables-main-container {
+	background-color: var(--main-bg-color);
+	padding-top: 6rem;
+}
+.timetables-content-container {
+	display: flex;
+	flex-direction: row;
+
```
Comment (khaledkz) [inline client/src/pages/Timetables/TimetableNotRegistered.css:8]: can you remove empty line

File: client/src/pages/Timetables/TimetableNotRegistered.css [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+	max-width: 36.875rem;
+	border-radius: 5%;
+	position: absolute;
+	left: 8%;
+	top: -30%;
+}
+
+.bottom-container {
+	background-color: var(--main-bg-color);
+
```
Comment (khaledkz) [inline client/src/pages/Timetables/TimetableNotRegistered.css:92]: same here

File: client/src/pages/Timetables/TimetableNotRegistered.css [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+		border-radius: 5%;
+		position: absolute;
+		left: 50px;
+		top: -110px;
+	}
+}
+@media screen and (min-device-width: 1200px) and (max-device-width: 1300px) {
+	.bottom-container {
+		background-color: var(--main-bg-color);
+
```
Comment (khaledkz) [inline client/src/pages/Timetables/TimetableNotRegistered.css:162]: same here

File: client/src/pages/Timetables/TimetableNotRegistered.css [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+		column-gap: 0rem;
+	}
+	.rightImg img {
+		max-width: 200px;
+		max-height: 100px;
+	}
+	.timestables-text-container {
+		width: 30%;
+	}
+}
```
Comment (khaledkz) [inline client/src/pages/Timetables/TimetableNotRegistered.css:255]: remove all empty lines

File: client/src/pages/Timetables/TimetableNotRegistered.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+						src={arrowImg}
+						alt="an arrow forming a heart"
+					/>
+					<img
+						className="rightImg"
+						src={rightImg}
+						alt="timetable illustration"
+					/>
+				</section>
+			</main>
```
Comment (khaledkz) [inline client/src/pages/Timetables/TimetableNotRegistered.js:74]: can you fix indentation

**Review by khaledkz — COMMENTED** [review]
looks good, try to review your PR and look to your code here after creating PR a lot of comments are the same about indentation and empty lines.

File: client/src/pages/Timetables/TimetableNotRegistered.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+					<section className=" timestables-text-container">
+						<p>
+							We offer FREE visual timetables with a clear structure, which can
+							help your child to settle into a routine. We have several
+							different types of visual timetables suitable for all children of
+							different ages.
+						</p>
+						<p>
+							To access online or download <br></br> FREE Timetable{" "}
+							<Link to="/login">Sign Up.</Link>
```
Comment (youssef910) [inline client/src/pages/Timetables/TimetableNotRegistered.js:26]: add the right link to login page

**Review by youssef910 — CHANGES_REQUESTED** [review]

File: client/src/pages/Timetables/TimetableNotRegistered.js (hunk 6 above)
Comment (andrejskalnins) [reply client/src/pages/Timetables/TimetableNotRegistered.js:26]: Hi, is the correct path /getstarted?

Comment (andrejskalnins) [reply client/src/pages/Timetables/TimetableNotRegistered.js:26]: Oh nwm spotted it

**Pull request merged**